packages/devtools_app/lib/src/screens/memory/panes/control/adb_button.dart

```diff
@@ -0,0 +1,43 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
we do not need separate files for each widget class. It is Dart common practice to have multiple classes in a single file. We should consolidate several of these files into one or into a couple, as we don't want to start the precedent of putting each class in a new file, like Java for example.

Reply:
discussed offline